app/helpers/application_helper.rb

```diff
@@ -126,12 +121,7 @@ def follow_button(followable, style = "full", classes = "")
       name: :button,
       type: :button,
       data: {
-        info: {
-          id: followable.id,
-          className: followable_type,
-          name: followable_name,
-          style: style
-        }
+        info: DataInfo.to_json(object: followable, className: followable_type, style: style)
```

Comment:
Technically not necessary, but the `DataInfo.to_json` is helping produce consistent attributes in Rails side code.  So why not re-use it.